src/eq_schema/RoutingConditions.js

```diff
@@ -14,11 +14,15 @@ class RoutingConditions {
       const optionIds = map("id", condition.answer.options);
       const notIds = xor(condition.routingValue.value, optionIds);
 
-      return notIds.map(id => ({
+      const whenArray = notIds.map(id => ({
```

Comment:
could just chain a concat on: `return notIds.map(foo).concat({})`?

Reply:
Also, i know i wrote this, but perhaps we could rename `notIds` to `unselectedIds` or something?